tests/instruments/keysight/test_keysightDSOX1102G_expectedSetup.txt

```diff
@@ -0,0 +1,3717 @@
+#800087683<setup prod="InfiniiVision" ver="01.01.2016092800" srver="2.0">
```

Comment:
Where is this file used in the test? I could not find this filename elsewhere...

Reply:
Good catched - used in previous iteration, not anymore. Will delete it.